ClipboardItem/ClipboardItem_ClipboardItem_interface.html

```diff
@@ -0,0 +1,132 @@
+  <dd>Returns an {{jsxref("Array")}} of MIME types available within the <strong><code>ClipboardItem</code></strong></dd>
+  <!-- OR -->
+  <!-- <dd>Returns an {{jsxref("Array")}} of MIME types supported by <code>ClipboardItem</code></dd> -->
+</dl>
+
+<h2 id="Methods">Methods</h2>
+
+<p><em>This interface defines the following methods.</em></p>
+
+<dl>
+ <dt>{{domxref("ClipboardItem.getType", "getType()")}}</dt>
+ <dd>Returns the {{Glossary("MIME type")}} of the file {{domxref("Blob")}} within the respective <strong><code>ClipboardItem</code></strong></dd>
+</dl>
+
+<h2 id="Examples">Examples</h2>
+
+<!-- the *clipboard* landing page should probably include a thorough example, it currently does not, hence why these are brief for now -->
+
+<h3 id="Writing_To_The_Clipboard">Writing To Clipboard</h3>
+
+<p>Here we're writing a new {{domxref("ClipboardItem.ClipboardItem()")}} to the {{domxref("Clipboard API"), "clipboard"}} by requesting a png image using the {{domxref("Fetch API")}}, and in turn, the {{domxref("Body.blob()"), responses' blob()}} method, to create the new {{domxref("ClipboardItem")}}.</p>
+
+<pre class="brush: js">
+async function writeClipImg() {
+  try {
+    const imgURL = '/myimage.png';
+    const data = await fetch(imgURL);
+    const blob = await data.blob();
+
+    await navigator.clipboard.write([
+      new ClipboardItem({
+        [blob.type]: blob
+      })
+    ]);
+    console.log('Fetched image copied.');
+  } catch(err) {
+    console.error(err.name, err.message);
+  }
+}
+</pre>
+
+<h3 id="Reading_From_The_Clipboard">Reading From The Clipboard</h3>
+
+<p>Here we're returning all items on the clipboard via the {{domxref("clipboard.read()")}} method. Then utilizing the {{domxref("ClipboardItem.types")}} property to set the {{domxref("ClipboardItem.getType", "getType()")}} argument and return the corresponding blob object.</p>
+
+<pre class="brush: js">
+async function getClipboardContents() {
+  try {
+    const clipboardItems = await navigator.clipboard.read();
+
+    for (const clipboardItem of clipboardItems) {
+      
+      for (const type of clipboardItem.types) {
+        const blob = await clipboardItem.getType(type);
+        // we can now use blob here
+      }
+
+    }
+
+  } catch (err) {
+    console.error(err.name, err.message);
+  }
+}
+</pre>
+
+<div class="note">
+  <p><strong>Note</strong>: At this time, only support for 'image/png' has been implemented in Chrome, with support for more file types planned for the future. To work with text see the {{domxref("Clipboard.readText()")}} and {{domxref("Clipboard.writeText()")}} methods of the {{domxref("Clipboard")}} interface.</p>
+</div>
+
+<div class="note">
+  <p><strong>Note</strong>: To read and write to the {{domxref("Clipboard API")}} you need to set the correct permissions via the {{domxref("Permissions API")}}. The {{domxref("Clipboard.read()")}} and {{domxref("Clipboard.write()")}} pages have good examples of this.</p>
+</div>
```

Comment:
This is primary information about how the API works, so move it to the introduction. Then add this information in appropriate locations.

The `clipboard-write` permission is granted automatically to the active tab. The browser requests the  `clipboard-read`, I think, when read() is called or when the script requests it directly. I've sent an email requesting clarification on this point.

Reply:
I've added this info to all the existing pages too